### DISCUSSION TITLE
Tracking failed salt states count in grafana

### DISCUSSION THREAD

**Discussion by petiepooo** [Security-Onion-Solutions/securityonion#7871] [Ideas] [answered]
Just a thought.. it could be useful to track failed salt states in grafana.  It would involved parsing the output of the periodic highstate runs for the failed number and pushing that into influx, but a little gauge in the overview area that would light up red with any failures would be a great first-glance indicator that something is wrong.

**Comment by dougburks** [comment, marked as answer]
We may end up doing something like this directly in SOC rather than Grafana.
